packages/web-frontend/src/selectors/dashboardSelectors.js

```diff
@@ -17,7 +22,37 @@ export const selectCurrentDashboardGroupCode = createSelector(
       : Object.keys(dashboardConfig)[0],
 );
 
+export const selectCurrentDashboardGroupId = createSelector(
+  [state => state.global.dashboardConfig, selectCurrentDashboardGroupCode],
+  (dashboardConfig, currentDashboardGroupCode) => {
+    const dashboardGroup = dashboardConfig[currentDashboardGroupCode];
+    if (!dashboardGroup) return null;
+    return Object.values(dashboardGroup)[0].dashboardGroupId;
```

Comment:
```suggestion
    return dashboardGroup ? Object.values(dashboardGroup)[0].dashboardGroupId : null;
```

---

Also, I am surprised we just select the first value in `dashboardGroup`, but it seems this was the existing alogrithm. By looking at the state, it seems that the keys for `dashboardGroup` values are permission groups:
```javascript
const state = {
  global: {
    dashboardConfig: {
      General: {
        Public: { dashboardGroupId: 155 },
        Donor: { dashboardGroupId: 156 },
        Admin: { dashboardGroupId: 157 },
      },
    },
  },
};
```

Any idea what's happening here?

Reply:
Frankly, I just implemented the existing algorithm and tested that it worked, so I'm not too sure. For time's sake, I tried to keep the changes to a minimum.

Reply:
I agree, and that's a good approach! I was just wondering whether you happen to have any idea about what's going on here.

If not, no worries! We can worry about this another day.

Reply:
Let's do that 👍